lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -117,14 +117,30 @@ def instructor_dashboard_2(request, course_id):  # lint-amnesty, pylint: disable
 
     course = get_course_by_id(course_key, depth=None)
 
+    permission_set = [
```

Comment:
How about changing permission_set to permissions or permission_list, I think the set suffix could be misunderstood.